### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

**Comment by ryonakano** [comment]
Hi @Vistaus, thank you for your translation! Actually I changed one text in the appdata in #10 just at the same time you create this PR. I merged #10 and re-updated your translation files so could you revisit the `po/extra/nl.po` and add a translation against the new string? :pray:

File: po/extra/nl.po [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+#: data/louper.appdata.xml.in:16
+msgid "Select some text"
+msgstr "Selecteer tekst"
+
+#: data/louper.appdata.xml.in:17
+msgid ""
+"Launch the app (it is recommended to assign a shortcut key in Keyboard "
+"Settings). The app shows the selected text in huge size"
+msgstr ""
```
Comment (ryonakano) [inline po/extra/nl.po:45]: FYI this is where I'd like you to add a translation :)

Comment (Vistaus) [reply po/extra/nl.po]: Done! 🙂

**Review by ryonakano — APPROVED** [review]
LGTM, thank you for your translation as always! 😄

**Comment by Vistaus** [comment]
> LGTM, thank you for your translation as always!

My pleasure! 🙂

**Pull request merged**